code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +165,26 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone

```

Comment:
it does, the saxophone code just creates its own variable instead of overriding the previous one for some reason. i'll change that.